apps/bublik/types/mdx.d.ts

```diff
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /* SPDX-FileCopyrightText: 2021-2023 OKTET Labs Ltd. */
 declare module '*.mdx' {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Commented out code - why?

Reply:
When we upgraded linter is became a bit more strict to not allow `any` type to be used

Since we are importing `.mdx` file into ts file it compiles as a default module export of markdown content with JSX
We are not using anything for meta handling of markdown files such as frontmatter so I just disabled any checks for this
We are not expected to pass anything to this component but it accepts some default props so we just ignore it

It's used only for auto-generated changelog